### DISCUSSION TITLE
Invalid format of the template extra arguments

### DISCUSSION THREAD

**Discussion by dmitrygribenchuk** [semaphoreui/semaphore#1063] [Q&A]
Hello. Have a strange issue (?) with CLI args in the template. I try to specify a variable in each template.

When I use invalid JSON - the internal validator solves it:
![image](https://user-images.githubusercontent.com/15670247/196650033-d6400df1-a6c9-4239-b33d-3ef230e67d51.png)

When I use stringified JSON I can save data and have an error when running a task:

![image](https://user-images.githubusercontent.com/15670247/196652707-f78f7416-c795-4576-8994-0a4436658914.png)

I use a stringified json.
`"{\"office_tag\": \"offvno\"}"`

**Comment by Richardswe** [comment]
It's just the wrong format? 
You should use brackets [ ] , example:
`[
  "-u",
 "admin"
]`
with the ]  bracket on the last row.
